server/services/store/sqlstore/migrations/bindata.go

```diff
@@ -1,446 +1,208 @@
-// Code generated by go-bindata. DO NOT EDIT.
```

Comment:
This file is no longer generated by `go-bindata`?

Reply:
It is. For me, go-bindata itself removed this comment. Not sure why though.